ui/app/scripts/app.js

```diff
@@ -3,19 +3,20 @@
 /**
  * The application.
  */
-var app = angular.module('uiApp', [
+angular.module('uiApp', [
   'ngResource',
   'ngMessages',
   'ngCookies',
   'ui.router',
   'mgcrea.ngStrap',
-  'satellizer'
-]);
+  'satellizer',
+  'ngMaterial'
+])
 
 /**
  * The run configuration.
  */
-app.run(function($rootScope) {
+.run(function($rootScope, $state) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['$state' is defined but never used.](https://www.codacy.com/app/HiP-App/HiPCMS/file/2288686120/issues/source?bid=3144120&fileBranchId=3183264#l19)